build/Targets/VSL.Versions.targets

```diff
@@ -73,7 +73,7 @@
   </Choose>
 
   <!-- Returns the current build version. Used in .vsixmanifests to substitute our build version into them -->
-  <Target Name="GetBuildVersion" Outputs="$(BuildVersion)" />
+  <Target Name="GetBuildVersion" Outputs="$(RoslynSemanticVersion).$(BuildNumberPart1.Trim())$(BuildNumberPart2.Trim())" />
```

Comment:
Why the Trim()s? Shouldn't `BuildNumberPart1` and `BuildNumberPart2` come pre-trimmed?